pagopa/io-app
# fix: [IA-71] Wrong secondary button in choose payment method screen

## Short description
This PR fix a bug in the `PickPaymentMethodScreen` component: the secondary button has the wrong label and clicking on it doesn't send the user to the previous screen but to the transaction summary screen.

## List of changes proposed in this pull request
- Dispatch a navigate back instead of a navigate to transaction summary screen.
- Modify the wrong label

https://user-images.githubusercontent.com/11773070/123655018-e959be00-d82e-11eb-81e5-5632f4e8b50f.mp4


## How to test
Start a payment, click on the `Change method` button, than click on `Back` button. 
You should return on the previous screen.
